src/messages/pt.json

```diff
@@ -241,21 +335,21 @@
   "support_status.title": "Status",
   "support_status.description": "Acompanhe o status da plataforma e saiba o histórico de instabilidade.",
   "support_plans.title": "Planos de suporte",
-  "support_plans.link": "/faq/como-funciona-o-suporte-da-vtex",
   "support_plans.description": "Veja detalhes dos planos de suporte disponíveis para cada plano comercial.",
+  "announcement_bar.headline": "📢 Queremos saber mais sobre você e como você utiliza a nossa documentação. ",
+  "announcement_bar.button": "Participe de nossa pesquisa! Leva menos de 5 minutos.",
   "documentation_reading_time.text": "{minutes} min de leitura",
   "sidebar_known_issues.description": "Acesse conteúdo e encontre soluções para todas as suas dúvidas sobre a plataforma.",
   "sidebar_support_rules.description": "Veja detalhes dos planos de suporte disponíveis para cada plano comercial.",
-  "updates_info_notes.title": "Perguntas frequentes",
+  "updates_info_notes.title": "Perguntas Frequentes",
   "updates_info_notes.description": "Encontre respostas para suas principais dúvidas de forma rápida e prática.",
   "menu_status.description": "Acompanhe o registro do histórico de plataforma.",
   "sidebar_support_rules.title": "Diretivas de suporte",
-  "sidebar_support_rules.link": "/faq/como-funciona-o-suporte-da-vtex",
-  "sidebar_known_issues.title": "Problemas conhecidos",
+  "sidebar_known_issues.title": "Problemas Conhecidos",
```

Comment:
why change this? please revert

Reply:
This change is due to another modification occurring after the creation of this branch. I will restore the original information.